### DISCUSSION TITLE
Tensorboard logs not being written when using sqlite storage

### DISCUSSION THREAD

**Discussion by gameveloster** [optuna/optuna#4418] [Q&A] [answered]
When I switch to using a study created with `storage="sqlite:///study.db"`, the stable_baseline creates the tensorboard log files but does not log to them.

**Previous study that has tensorboard logs being written**
```py
study = optuna.create_study(
    pruner=optuna.pruners.SuccessiveHalvingPruner(
        min_resource="auto",
        reduction_factor=4,
    ),
)
study.optimize(optimize_agent, n_trials=10, n_jobs=2)
```

**New study that has no tensorboard logs written**
```py
study = optuna.create_study(
    study_name='study',
    storage="sqlite:///study.db",
    pruner=optuna.pruners.SuccessiveHalvingPruner(
        min_resource="auto",
        reduction_factor=4,
    ),
)
study.optimize(optimize_agent, n_trials=10, n_jobs=2)
```

Is there an additional config to set so that the tensorboard summary writer will write the logs to the tensorboard log files? Thanks

**Comment by nzw0301** [comment, marked as answer]
Hi, could you share minimal reproducible codes with us?

**Comment by gameveloster** [comment]
Fixed! Turns out to be a problem with tensorboard.
